documentation/modules/metrics/ref_metrics-config-files.adoc

```diff
@@ -5,50 +5,114 @@
+│   ├── strimzi-cruise-control.json
+│   ├── strimzi-kafka-bridge.json
 │   ├── strimzi-kafka-connect.json
-│   ├── strimzi-kafka.json
-│   ├── strimzi-zookeeper.json
+│   ├── strimzi-kafka-exporter.json <2>
 │   ├── strimzi-kafka-mirror-maker-2.json
+│   ├── strimzi-kafka.json
 │   ├── strimzi-operators.json
-│   ├── strimzi-kafka-bridge.json
-│   ├── strimzi-cruise-control.json
-│   └── strimzi-kafka-exporter.json <3>
-├── kafka-connect-metrics.yaml <4>
-├── kafka-metrics.yaml <5>
-├── kafka-mirror-maker-2-metrics.yaml <6>
-├── kafka-bridge-metrics.yaml <7>
-├── kafka-cruise-control-metrics.yaml <8>
+│   └── strimzi-zookeeper.json
+├── grafana-install
+│   └── grafana.yaml <3>
 ├── prometheus-additional-properties
-│   └── prometheus-additional.yaml <9>
+│   └── prometheus-additional.yaml <4>
 ├── prometheus-alertmanager-config
-│   └── alert-manager-config.yaml <10>
-└── prometheus-install
-    ├── alert-manager.yaml <11>
-    ├── prometheus-rules.yaml <12>
-    ├── prometheus.yaml <13>
-    ├── strimzi-pod-monitor.yaml <14>
+│   └── alert-manager-config.yaml <5>
+├── prometheus-install
+│    ├── alert-manager.yaml <6>
+│    ├── prometheus-rules.yaml <7>
+│    ├── prometheus.yaml <8>
+│    ├── strimzi-pod-monitor.yaml <9>
+├── kafka-bridge-metrics.yaml <10>
+├── kafka-connect-metrics.yaml <11>
+├── kafka-cruise-control-metrics.yaml <12>
+├── kafka-metrics.yaml <13>
+└── kafka-mirror-maker-2-metrics.yaml <14>
 --
-<1> Installation file for the Grafana image
-<2> Grafana dashboards
-<3> Grafana dashboard specific to xref:assembly-kafka-exporter-{context}[Kafka Exporter]
-<4> Metrics configuration that defines Prometheus JMX Exporter relabeling rules for Kafka Connect
-<5> Metrics configuration that defines Prometheus JMX Exporter relabeling rules for Kafka and ZooKeeper
-<6> Metrics configuration that defines Prometheus JMX Exporter relabeling rules for Kafka Mirror Maker 2.0
-<7> Kafka Bridge resource with metrics enabled
-<8> Metrics configuration that defines Prometheus JMX Exporter relabeling rules for Cruise Control
-<9> Configuration to add roles for service monitoring
-<10> Hook definitions for sending notifications through Alertmanager
-<11> Resources for deploying and configuring Alertmanager
-<12> Alerting rules examples for use with Prometheus Alertmanager (deployed with Prometheus)
-<13> Installation file for the Prometheus image
-<14> Prometheus job definitions to scrape metrics data from pods
+<1> Grafana dashboards
+<2> Grafana dashboard specific to xref:assembly-kafka-exporter-{context}[Kafka Exporter]
+<3> Installation file for the Grafana image
+<4> Configuration to add roles for service monitoring
+<5> Hook definitions for sending notifications through Alertmanager
+<6> Resources for deploying and configuring Alertmanager
+<7> Alerting rules examples for use with Prometheus Alertmanager (deployed with Prometheus)
+<8> Installation file for the Prometheus image
+<9> Prometheus job definitions to scrape metrics data from pods
```

Comment:
Actually they are `PodMonitor` definitions for the Prometheus operator that translates them to Prometheus job for its configuration. I would change somehow to say this and not letting people to expect to see "job" definitions in this file.

Reply:
Changed to:

> PodMonitor definitions that translate to jobs for the Prometheus operator to be able to scrape metrics data directly from pods.